microsetta_public_api/api/microsetta_public_api.yml

```diff
@@ -1114,6 +1114,27 @@ paths:
         '404':
           $ref: '#/components/responses/404NotFound'
 
+  '/dataset/{dataset}/taxonomy/empress/{resource}':
+    parameters:
+      - $ref: '#/components/parameters/namedDataset'
+      - $ref: '#/components/parameters/taxonomyResource'
+    get:
+      operationId: microsetta_public_api.api.taxonomy.get_empress
+      tags:
+        - Taxonomy
+        - Plotting
+      summary: Get the empress object corresponding to the taxonomy for a given dataset.
+      description: Get the empress object corresponding to the taxonomy for a given dataset.
+      responses:
+        '200':
+          description: An empress object.
+          content:
+            application/json:
+              schema:
+                type: object
```

Comment:
I realize it may be helpful to not describe the schema at the moment, but would it be possible to expand the description about how someone may get more information on consuming data from this endpoint?

Reply:
https://github.com/biocore/microsetta-public-api/pull/84/commits/4e09fc1f1350d206920b5c38ff083a7f7c3951b3